docker/scripts/mininet/multi_switch_mininet.py

```diff
@@ -79,9 +81,11 @@ def main():
     conf = manifest['targets'][args.target]
     params = conf['parameters'] if 'parameters' in conf else {}
 
+    os.environ.update(dict(map(lambda (k,v): (k, str(v)), params.iteritems())))
+
     def formatParams(s):
         for param in params:
-            s = s.replace('%'+param+'%', str(params[param]))
+            s = s.replace('{'+param+'}', str(params[param]))
```

Comment:
Yes, we still need our own formatting for formatting strings other than commands (for example, formatting latencies). I was thinking of doing `$foo` everywhere, but it's not a very python-y way of formatting strings. There are some places where using this is messy (or even problematic). For example, formatting the string: `"{latency}ms"` with the dollar is `"$latencyms"`. However, I think it would be good to have the same dollar syntax everywhere for consistency. What do you think? Should we just do `$foo` everywhere?